lineman/app/models/user_records_interface.coffee

```diff
@@ -5,6 +5,8 @@ angular.module('loomioApp').factory 'UserRecordsInterface', (BaseRecordsInterfac
     constructor: (recordStore) ->
       @baseConstructor recordStore
       @profileClient = new RestfulClient 'profile'
+      @profileClient.onSuccess = (response) =>
```

Comment:
This is a red flag for me; means we've set up our client / interface relationship a little odd.